packages/host/app/components/operator-mode/detail-panel-selector.gts

```diff
@@ -0,0 +1,202 @@
+                class={{cn
+                  'boxel-selector__item'
+                  boxel-selector__item--selected=selectorItem.selected
+                  boxel-selector__item--disabled=selectorItem.disabled
+                }}
+                data-test-boxel-selector-item
+                data-test-boxel-selector-item-selected={{selectorItem.selected}}
+              >
+                {{! template-lint-disable require-context-role }}
+                <div
+                  class='boxel-selector__item__content'
+                  role='menuitem'
+                  href='#'
+                  data-test-boxel-selector-item-text={{selectorItem.text}}
+                  {{on
+                    'click'
+                    (fn this.invokeSelectorItemAction selectorItem.action)
+                  }}
+                  {{on
+                    'keypress'
+                    (fn this.invokeSelectorItemAction selectorItem.action)
+                  }}
+                  disabled={{selectorItem.disabled}}
+                >
+                  <span class='selector-item'>
+                    {{selectorItem.text}}
+                  </span>
+                </div>
+              </li>
+            </:item>
+          </SelectorItemRenderer>
+        {{/each}}
+      {{/if}}
+    </ul>
+    <style>
+      @layer {
+        .boxel-selector {
+          --boxel-selector-border-radius: var(--boxel-border-radius);
+          --boxel-selector-color: var(--boxel-light);
+          --boxel-selector-current-color: var(--boxel-light-100);
+          --boxel-selector-selected-color: var(--boxel-highlight);
+          --boxel-selector-disabled-color: var(--boxel-highlight);
+          --boxel-selector-font: 500 var(--boxel-font-sm);
+          --boxel-selector-item-gap: var(--boxel-sp-xxs);
+          --boxel-selector-item-content-padding: var(--boxel-sp-xs)
+            var(--boxel-sp);
+          --boxel-selector-selected-background-color: var(--boxel-highlight);
+          --boxel-selector-selected-font-color: var(--boxel-light-100);
+          --boxel-selector-selected-hover-font-color: var(--boxel-dark);
+          list-style-type: none;
+          margin: 0;
+          padding: 0;
+          background-color: var(--boxel-selector-color);
+          border-radius: var(--boxel-selector-border-radius);
+        }
+
+        .boxel-selector__item {
+          font: var(--boxel-selector-font);
+          letter-spacing: var(--boxel-lsp-sm);
+          border-radius: inherit;
+        }
+
+        .boxel-selector__item--selected {
+          background-color: var(--boxel-selector-selected-background-color);
+          color: var(--boxel-selector-selected-font-color);
+        }
+
+        .boxel-selector__item--selected:hover {
+          color: var(--boxel-selector-selected-hover-font-color);
+        }
+
+        .boxel-selector__item:hover {
+          background-color: var(--boxel-selector-current-color);
+          cursor: pointer;
+        }
```

Comment:
Place the `.boxel-selector__item--selected:hover` styles after this block so that the selected-hover state has the different background-color I mentioned above.